libero-config/config--libero-default.js

```diff
@@ -2,13 +2,13 @@ const Color = require('color');
 
 const config = { data: {}, layerAllocations: {} };
 
-config.data.baselinegrid = {space: {}};
-config.data.baselinegrid.space.extra_small_in_px = 12;
-config.data.baselinegrid.space.small_in_px = '!expression baselinegrid.space.extra_small_in_px * 2';
-config.data.baselinegrid.space.smallish_in_px = '!expression baselinegrid.space.small_in_px * 1.5';
-config.data.baselinegrid.space.medium_in_px = '!expression baselinegrid.space.small_in_px * 2';
-config.data.baselinegrid.space.large_in_px = '!expression baselinegrid.space.small_in_px * 3';
-config.data.baselinegrid.space.extra_large_in_px = '!expression baselinegrid.space.small_in_px * 5';
+config.data.baseline_grid = {space: {}};
```

Comment:
I think this should be kept as `baselinegrid` to make the generated sass variables easier to read.

Reply:
Wonder if `_` should become `-`?

Reply:
I think the alternatives we have for the ultimate sass variable name format are:
`$baselinegrid-space-extra_small_in_px: 12;  // 1`
or
`$baseline_grid-space-extra_small_in_px: 12;  // 2`
or
`$baseline-grid-space-extra_small_in_px: 12;  // 3`

I prefer 1 over 2 because I think it reads better, even though 2 is more strictly consistent.

I don't favour 3 because it confuses the issue at the config end where we output a hyphen into the sass name to represent a boundary between respective parent and child objects.

Reply:
I meant everywhere, so `$baseline-grid-space-extra-small-in-px` (but still reads a bit funny). I think all of these suffer from being quite wordy.

Preference for 2 for correctness (it is two words).